app/javascript/packs/pw-strength.js

```diff
@@ -130,4 +130,4 @@ function analyzePw() {
   });
 }
 
-document.addEventListener('DOMContentLoaded', analyzePw);
+document.addEventListener('keydown', analyzePw);
```

Comment:
What was the reason for this change? I see in the code block above we bind an `'input'` event listener to the input, so I would expect we were already re-computing strength when the value changes. This could also make it so we're adding a new input event listener to the field every time the user presses a key, and those keypresses also wouldn't be limited to the specific field.

Reply:
The intent here was that UX wanted the password strength bar to appear only when there are characters in the `input` field. With this change, I want to start computing password strength as the user is typing.

Reply:
For the event listener, I think that having the event listener fire once per event could be the solution--this way we aren't adding additional listeners and we are still computing the password strength.

Reply:
Resolving this, but for posterity: Jessica and I paired on this and came to an alternative solution in b480236 to avoid the additional event handlers.